src/http/routes/grant-type/grant-type.spec.js

```diff
@@ -22,6 +22,28 @@ describe('Grant Type Tests', () => {
     id: 'example-grant'
   };
 
+  const pageVariables = {
```

Comment:
Thanks, I was going to do this. We might need to break it into a 'mocks' file at some point.

Reply:
Not sure about that, I'm not keen on these tests at all. They're very brittle and testing detail rather than behavior.